Automattic/wp-calypso
# Signup: Remove free trails from the plans only signup step

This removes free trials from the `/plans` signup flow.

#### Testing instructions

1. Run `git checkout remove/free-trials-from-plans-only-flow` and start your server
2. Open the [`Plan signup` page](http://calypso.localhost:3000/start/plan)
3. Check that on the select plan you are not offered a free trial
4. Check that you can still sign up for a paid plan.

#### Reviews
